pki.spec

```diff
@@ -902,16 +890,6 @@ pkgs=base\
     --install-dir=%{buildroot} \
     install
 
-# Unbundle the FontAwesome fonts
-rm %{buildroot}%{_datadir}/pki/common-ui/fonts/fontawesome-webfont.woff
-%if 0%{?fedora} > 38
-ln -s ../../../fonts/fontawesome4/fontawesome-webfont.woff \
-    %{buildroot}%{_datadir}/pki/common-ui/fonts/fontawesome-webfont.woff
-%else
-ln -s ../../../fonts/fontawesome/fontawesome-webfont.woff \
-    %{buildroot}%{_datadir}/pki/common-ui/fonts/fontawesome-webfont.woff
-%endif
-
```

Comment:
I'm not sure about removing this code. To my understanding this came from Fedora:

* https://src.fedoraproject.org/rpms/dogtag-pki/c/94475fdcf3ba2f1ab82fcc18a6838bb7dd1eb84f
* https://github.com/dogtagpki/pki/commit/56efe4af357ed646425742ef46b5733ce4b405a8

and we do have references to `fontawesome-webfont` in `patternfly.css`.

Reply:
OK, I see that https://github.com/dogtagpki/pki/commit/56efe4af357ed646425742ef46b5733ce4b405a8 was breaking the CI, probably because that code only works if we build the theme package (which is not always the case). For reverting a breaking change I think you can just go ahead and merge it, no need to use a PR.

Reply:
Apparently it's breaking the CI for PRs too so I went ahead and merge the commit that reverts https://github.com/dogtagpki/pki/commit/56efe4af357ed646425742ef46b5733ce4b405a8.